bbc_restrictions/views/mrp_view.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+
+		<record model="ir.ui.view" id="bbc_restrictions_mrp_bom_form_view">
+			<field name="name">mrp.bom.form.restriction.inherit</field>
+			<field name="model">mrp.bom</field>
+        	<field name="inherit_id" ref="mrp.mrp_bom_form_view"/>
+        	<field name="arch" type="xml">
+        		<xpath expr="/form/group/group/field[@name='product_id']" position="attributes">
+        			<attribute name="options">{'no_create': true}</attribute>
+        		</xpath>
+        		<xpath expr="/form/group/group/field[@name='product_tmpl_id']" position="attributes">
+        			<attribute name="options">{'no_create': true}</attribute>
+        		</xpath>
+                <xpath expr="/form/group/group/div/field[@name='product_uom']" position="attributes">
+                    <attribute name="options">{'no_create': true}</attribute>
+                </xpath>
+        		<xpath expr="/form/notebook/page[@string='Components']//field[@name='product_id']" position="attributes">
```

Comment:
The indentation is still off

Reply:
You're right! I was too soon with my commit and did not check the other views.
I installed a tool in my editor now that checks the XML indentation for each XML-file automatically. I re-indent every XML-file with this tool.
What you'll see is that all XML code is displayed without white lines. Do you prefer these white lines for easy reading? Or is the XML code OK as it is in my last commit?

Reply:
Not my preference, but it's fine really. If you remove the &quot; in the domains we can proceed.